jenkins/Jenkinsfile

```diff
@@ -1,19 +1,18 @@
 def configuration = [vaultUrl: "${VAULT_URL}",  vaultCredentialId: "vault-app-role", engineVersion: 2]
 
 def secrets = [
-  [path: 'secret/jenkins/cloudify', engineVersion: 2, secretValues: [
-    [envVar: 'OS_USERNAME', vaultKey: 'username'],
-    [envVar: 'OS_TENANT', vaultKey: 'tenant'],
-    [envVar: 'OS_IP', vaultKey: 'ip'],
-    [envVar: 'OS_PASSWORD', vaultKey: 'password']]],
+  [path: 'secret/jenkins/cloudifyaws', engineVersion: 2, secretValues: [
+    [envVar: 'MANAGER_USERNAME', vaultKey: 'username'],
+    [envVar: 'MANAGER_TENANT', vaultKey: 'tenant'],
+    [envVar: 'MANAGER_IP', vaultKey: 'ip'],
+    [envVar: 'MANAGER_PASSWORD', vaultKey: 'password']]],
 ]
 
 @Library('pipeline-shared-library') _
 
 pipeline {
   agent {
     kubernetes {
-      label "cloudify-agent-${env.BUILD_NUMBER}"
```

Comment:
I thought this was added to work around a problem- why is it removed for his?

Reply:
@yuvlar I think you did it for a reason right ?

Reply:
because this method is deprecated, 
when not specifying a label, it'll automatically create a random label to the agent, with the project name, branch and build number